tests/unity/wrap_test/src/fns/fns.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/*
+	DO NOT FORMAT THIS FILE!
+
+	It contains function declarations that is used to validate the test wrapper
+	scripts (func_name_list.py and header_parse.py) which renames the functions.
+	The regular expression has had several bugs in the past, and the formating
+	of the functions here are used to validate that	the fixes still work when
+	modifying the regex.
+
+	List of fixes this file tries to reproduce:
+	https://github.com/nrfconnect/sdk-nrf/pull/1413
+	https://github.com/nrfconnect/sdk-nrf/pull/2182
+	https://github.com/nrfconnect/sdk-nrf/pull/3432
+	https://github.com/nrfconnect/sdk-nrf/pull/5069
+	https://github.com/nrfconnect/sdk-nrf/pull/5814
+	https://github.com/nrfconnect/sdk-nrf/pull/6478
+	https://github.com/nrfconnect/sdk-nrf/pull/6547
+*/
+
+#ifndef __FNS_H
```

Comment:
Why the name fns?

Reply:
Just short for functions. Better alternatives are welcome :)

Reply:
what are you trying to achieve with the abbreviation, except confusing readers ?

`functions.h`
`test_functions.h` 
`wrap_functions.h`
are not that long so the deserve an abbreviation, and are all better options imho.

Reply:
Since this file not only has functions that are tested, perhaps it is better to name this `test_header_file.h`?

Reply:
It's not only a header file. What about _snippets_?

Reply:
so 
`snippets/snippets.c` and `snippets/snippets.h` ?

Fine by me

Reply:
not sure about snippets.

A snippet can also be something you might want to re-use and include elsewhere, which is not the case here.
Upstream `snippets` is proposed for common configuration: https://github.com/zephyrproject-rtos/zephyr/pull/40669
As well as used in linker scripts: https://github.com/zephyrproject-rtos/zephyr/blob/7dac931e3662eb666cdabc3bf1759a7484249730/include/arch/arm/aarch32/cortex_m/scripts/linker.ld#L136
```
#include <snippets-rom-start.ld>
```
so using snippet in this context may in future result in confusion as how the term snippet is used with Zephyr.
The word snippet itself are so broad that it may also cover how you use it here, just trying to ensure some consistency on how it's used in Zephyr/NCS.

Reply:
Sorry, I didn't see this comment. I don't have a strong opinion on this name, but I think anything is better than `foo`, which it was before. @lemrey is waiting for this fix to update libmodem, so it would be nice to conclude on a name soon. Suggestions are welcome.
How about just _code_ or _test_code_?

Reply:
Also fine by me

Reply:
Ok, I'll rename it to`test_code` :)